bedrock/careers/templates/careers/base.html

```diff
@@ -0,0 +1,76 @@
+{% set default_slug = "Mozilla Careers &mdash; Feel good about your work again" %}
+<!DOCTYPE html>
+<html lang="{{ LANG }}" dir="{{ DIR }}" class="no-js"
+      data-ga-code="{{ settings.GA_ACCOUNT_CODE }}"
+      data-gtm-code="{{ settings.GTM_ACCOUNT_CODE }}">
+  <head>
+    {# meta #}
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, minimum-scale=1.0, maximum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta property="og:type" content="website">
+    <meta property="og:title" content="{{ meta_title|default(default_slug, true)|safe }}">
+    <meta property="og:description" content="{{ meta_description|default(default_slug, true)|safe }}">
+    <meta property="og:image" content="https://careers.mozilla.org{{ meta_image|default(static('img/careers/moz-m.png'), true)|safe }}">
+    <meta property="og:image:alt" content="{{ meta_title|default(default_slug, true)|safe }}">
+    <meta name="twitter:title" content="{{ meta_title|default(default_slug, true)|safe }}">
+    <meta name="twitter:description" content="{{ meta_description|default(default_slug, true)|safe }}">
+    <meta name="twitter:image" content="https://careers.mozilla.org{{ meta_image|default(static('img/careers/moz-m.png'), true)|safe }}">
+    <meta name="twitter:card" content="summary">
+    <meta name="description" content="{{ meta_description|default(default_slug, true)|safe }}">
+    <title>{{ meta_title|default(default_slug, true)|safe }}</title>
+
+    {# css #}
+    {% block site_css %}
+      {{ css_bundle('careers') }}
+    {% endblock %}
+
+    {# js #}
+    <script src="{{ static('js/careers/dnt-helper.js') }}"></script>
+    <script src="{{ static('js/careers/protocol-base.min.js') }}"></script>
+    <script src="{{ static('js/careers/protocol-utils.min.js') }}"></script>
+    <script src="{{ static('js/careers/protocol-supports.min.js') }}"></script>
+
+    {# icons #}
+    <link rel="shortcut icon" type="image/png" href="{{ static('img/careers/favicon.ico') }}"/>
+
+    {# feeds #}
+    <link rel="alternate" type="application/rss+xml"
+          href="{{ url('careers.feed') }}" title="Current Mozilla job openings" />
+
+    {# TODO: Configure GA #}
+
+  </head>
+  <body id="{% block body_id %}{% endblock %}" class="{% block body_class %}{% endblock %}"
+    {%- block body_data %}
+    {%- endblock %}>
+    {%- if settings.GTM_ACCOUNT_CODE %}
+      <noscript>
+        <iframe src="//www.googletagmanager.com/ns.html?id={{ settings. GTM_ACCOUNT_CODE }}"
+                height="0" width="0" style="display:none;visibility:hidden"></iframe>
+      </noscript>
+    {%- endif %}
```

Comment:
Seeing as this is going to be switching domains, it probably makes sense for this to use bedrock's GTM setup?